lib/verify.js

```diff
@@ -7,9 +7,18 @@ module.exports = async (pluginConfig, { logger }) => {
     }
   }
   try {
-    await execa('docker', ['login', '-u=' + process.env.DOCKER_USERNAME, '-p=' + process.env.DOCKER_PASSWORD], {
-      stdio: 'inherit',
-    })
+    await execa(
+      'docker',
+      [
+        'login',
+        process.env.DOCKER_REGISTRY || '',
```

Comment:
Shouldn't this be a config option instead? Environment variables are really only used for credentials, because you wouldn't want to commit them to the repo but set them in CI

Reply:
Good point. I have updated this to be part of the verifyConditions config.